samples/HttpServer_AJAX/app/application.cpp

```diff
@@ -78,12 +78,40 @@ void onAjaxFrequency(HttpRequest &request, HttpResponse &response)
 	response.sendJsonObject(stream);
 }
 
+void onByteStream(HttpRequest &request, HttpResponse &response)
```

Comment:
It's basic exmaple, so I'm not sure what advanced byte streaming sample needed here.